bindings/gumjs/gumquickprocess.c

```diff
@@ -622,3 +635,61 @@ gum_quick_exception_handler_on_exception (GumExceptionDetails * details,
 
   return handled;
 }
+
+GUMJS_DEFINE_FUNCTION (gumjs_process_run_on_thread)
+{
+  GumQuickScope scope = GUM_QUICK_SCOPE_INIT (core);
+  GumThreadId thread_id;
+  JSValue user_func;
+  GumQuickRunOnThreadContext sync_ctx;
+  GumStalker * stalker;
+  gboolean success;
+
+  if (!_gum_quick_args_parse (args, "ZF", &thread_id, &user_func))
+    return JS_EXCEPTION;
+
+  if (thread_id == 0)
+    return JS_UNDEFINED;
+
+  _gum_quick_scope_suspend (&scope);
+
+  sync_ctx.core = core;
+  sync_ctx.scope = scope;
+  sync_ctx.user_func = JS_DupValue (core->ctx, user_func);
+  sync_ctx.sync = FALSE;
```

Comment:
This is unused now, right?

Reply:
You mean the `.sync` field is not needed, rather than the structure as a whole? I have removed it and removed other superfluous parameters and likewise for v8.